crypto/ocsp/ocsp_test.cc

```diff
@@ -1,5 +1,6 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The implementation currently supports a number of different OCSP response verification methods. Are we capturing all of them here?

Reply:
I should be more explicit. I was referring to the different verification methods that can be used to verify the OCSP response signature.
https://datatracker.ietf.org/doc/html/rfc6960#section-2.2

Reply:
Thanks, that's a really good callout that I didn't notice.
I'll add another test where the OCSP response is signed directly by the CA, instead of the OCSP responder.